lib/ast.js

```diff
@@ -142,6 +142,7 @@ var RenderT = module.exports.RenderT = new Enum( 'RenderT', {
 var Tex = module.exports.Tex = new Enum( 'Tex', {
     LITERAL: { args: [ RenderT ] }, // contents
     CURLY:   { args: [ ['self'] ] }, // expr
+    DOLLAR:  { args: [ ['self'] ] }, // expr
```

Comment:
I use this for rewriting `\begin{math}...\end{math}` to `$...$` as dollar signs cannot be used in the input and mhchem does not any other way to switch to math mode (that I could find) than the dollar sign